### PR TITLE
added document force https on apache behind nginx-lb

### PR DESCRIPTION
**Pull request by Tipset**

### CODE REVIEW

File: content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+weight: 1
+alwaysopen: true
+---
+
+## Edit httd.conf
+
+Add the following configuration in the apache configuration file `httd.conf`.
+
+
+```shell
```
Comment (MrMoreau) [inline content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md:13]:
```suggestion
```
```

Comment (MrMoreau) [inline content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md:13]: Maybe just use ```

File: content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Force HTTPS on Apache behind Nginx load balancer"
+description: "How to force HTTPS on apache with Nginx load balancer"
```
Comment (MrMoreau) [inline content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md:3]:
```suggestion
description: "How to force HTTPS on Apache with Nginx load balancer"
```

File: content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+weight: 1
+alwaysopen: true
+---
+
+## Edit httd.conf
+
+Add the following configuration in the apache configuration file `httd.conf`.
```
Comment (MrMoreau) [inline content/en/docs/jelastic-paas/guides/force_https_on_apache_behind_nginx.md:10]:
```suggestion
Add the following configuration in the Apache configuration file `httd.conf`.
```

**Review by MrMoreau — APPROVED** [review]
LGTM!

**Review by evgeluev — APPROVED** [review]
LGTM

**Pull request merged**